base/server/share/conf/tomcat.conf

```diff
@@ -9,7 +9,7 @@
 # Default NSS DB type is loaded from /usr/share/pki/etc/tomcat.conf
 
 # Where your java installation lives
-JAVA_HOME="[JAVA_HOME]"
+JAVA_HOME="[JRE_HOME]"
```

Comment:
@edewata Some of the time (via IPA installer) the install failed with unable to find `JRE_HOME` config value; from a line like:

```python
jre_home = self.instance.config['JRE_HOME']
```

-- does this mean I need to add JRE_HOME as a path here as well? I never saw `JAVA_HOME` in `/etc/pki/pki-tomcat/ca/CS.cfg`, only in `/etc/pki/pki-tomcat/tomcat.conf`, so that's the only place I could think of that `JRE_HOME` would be left.

Other times (pkispawn directly), the install would fail with JDK11-based Tomcat errors. That's what #525 fixes. 

Clearly, something is still being run under the default JDK, not the JDK we chose.

Reply:
This is visible here: https://github.com/dogtagpki/pki/pull/526/checks?check_run_id=1005012887#step:11:186